plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -188,7 +188,8 @@ func init() {
 			// glusterfs
 			rbac.NewRule("get", "list", "watch").Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
 			rbac.NewRule("get", "create", "delete").Groups(legacyGroup).Resources("services", "endpoints").RuleOrDie(),
-			rbac.NewRule("get").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
+			// glusterfs and Azure File
+			rbac.NewRule("create", "get").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
```

Comment:
Why does the azure file provisioner need to create a secret?  Isn't the point of attach/detach controller to be the one doing these operations so nodes *don't* have the permission?  Or are you saying the only way Azure file provisioning works is for the node to have highly privileged credentials (mount any file) and to do the operation from the node?